wodles/aws/aws_s3.py

```diff
@@ -2829,6 +2918,7 @@ def __init__(self, reparse, access_key, secret_key, aws_profile,
         self.remove_log_streams = remove_log_streams
         self.only_logs_after_millis = int(datetime.strptime(only_logs_after, '%Y%m%d').replace(
             tzinfo=timezone.utc).timestamp() * 1000) if only_logs_after else None
+        self.default_date_millis = int(self.default_date.timestamp()) * 1000
```

Comment:
It is more correct to multiply first, then cast to int, otherwise we could be losing up to 999 milis

Reply:
I totally overlooked that, solved in 4667473.